mitosheet/css/footer.css

```diff
@@ -36,10 +39,13 @@
 
 .footer-tab-bar {
     /* Deal with the overflow */
-    overflow-x: auto;
+    overflow: auto;
```

Comment:
Add comment explaining that for the scrollbar-gutter property to work, we need to set the overflow property. And since we don't want scrolling on the y axis, we can overwrite that one